Packet++/src/HttpLayer.cpp

```diff
@@ -534,8 +534,13 @@ static const std::unordered_map<int, HttpResponseStatusCode> intStatusCodeMap =
 	{599, HttpResponseStatusCode::Http599NetworkConnectTimeoutError},
 };
 
-HttpResponseStatusCode::HttpResponseStatusCode(const int &statusCodeNumber)
+HttpResponseStatusCode::HttpResponseStatusCode(const int &statusCodeNumber, const std::string statusMessage)
 {
+	if(statusMessage != "")
```

Comment:
it's `HttpResponseStatusCode` duty to handle the empty message, we don't do it in `HttpResponseFirstLine` anymore.